docs/source/examples/vision/checks/distribution/train_test_label_drift.ipynb

```diff
@@ -273,16 +273,16 @@
          "name": "Train Dataset",
```

Comment:
<p>This display (num per class) can't handle this number of classes. Need to add to it the same logic we have in metrics...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/877/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Let's open a different issue for that